numpy/add_newdocs.py

```diff
@@ -463,6 +463,45 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
```

Comment:
Don't think this will play well with numpydoc. Leaving the `see nditer` in the last set of arguments seems fine to me.